src/main/scala/fr/polytechnique/cmap/cnam/etl/events/MedicalAct.scala

```diff
@@ -59,3 +59,20 @@ object McoCIM10Act extends MedicalAct {
 object McoCEAct extends MedicalAct {
   val category: EventCategory[MedicalAct] = "mco_ce_act"
 }
+
+object SsrCEAct extends MedicalAct {
+  val category: EventCategory[MedicalAct] = "ssr_ce_act"
+}
+
+object SsrCCAMAct extends MedicalAct {
+  val category: EventCategory[MedicalAct] = "ssr_ccam_act"
+}
+
+
+object SsrCSARRAct extends MedicalAct {
```

Comment:
A small ScalaDoc to explain this would be helpful.

Reply:
Added one !